### PR TITLE
Fix(cohort builder, dashboard): center graphs in order to minimize wh…

### PR DESCRIPTION
**Pull request by evans-g-crsj**
…ite space

### CODE REVIEW

File: src/components/CohortBuilder/Summary/Summary.css [hunk 1]
```diff
@@ -51,3 +51,21 @@
   top: 0;
   bottom: 0;
 }
+
+.spinner {
```
Comment (francisl) [inline src/components/CohortBuilder/Summary/Summary.css:55]: the spinner is already managed, why we need another one?

File: src/components/UserDashboard/SavedQueries/SavedQueries.module.css [hunk 2]
```diff
@@ -81,3 +81,14 @@
   font-size: 12px;
   color: rgb(52, 52, 52);
 }
+
+.spinner {
```
Comment (francisl) [inline src/components/UserDashboard/SavedQueries/SavedQueries.module.css:85]: same comment, why we need a spinner when the Card is supposed to handle that

File: src/components/UserDashboard/index.js [hunk 3]
```diff
@@ -71,17 +58,23 @@ export default compose(
...
-            <MemberResearchInterestsChart />
+            <div style={{ height: '265px' }}>
+              <MemberResearchInterestsChart />
+            </div>
           </DashboardCard>
         </Col>
         <Col xs={24} sm={24} md={12} lg={12} xl={8}>
           <DashboardCard title="Most Frequent Diagnoses">
-            <MostFrequentDiagnosesChart />
+            <div style={{ height: '265px' }}>
```
Comment (francisl) [inline src/components/UserDashboard/index.js:75]: inline style?

Comment (evans-g-crsj) [reply src/components/UserDashboard/index.js:75]: good point

File: src/components/UserDashboard/SavedQueries/SavedQueries.module.css (hunk 2 above)
Comment (evans-g-crsj) [reply src/components/UserDashboard/SavedQueries/SavedQueries.module.css:85]: These cards are not antd cards

File: src/chartkit/components/ChartLoadGate.js [hunk 4]
```diff
@@ -1,22 +1,18 @@
...
 const ErrorText = () => <div>Error no data</div>;
 
-const ChartLoadGate = ({
-  fetchedState: { isLoading, data, error },
-  Chart,
-  Loader = LoadingSpinner,
-  Error = ErrorText,
-}) => (isLoading ? <Loader /> : data ? <Chart data={data} /> : <Error />);
+const ChartLoadGate = ({ fetchedState: { isLoading, data }, Chart, Error = ErrorText }) =>
+  isLoading ? <Spinner size={'large'} /> : data ? <Chart data={data} /> : <Error />;
```
[the remark this answers is not in the document]
Comment (francisl) [reply src/chartkit/components/ChartLoadGate.js:8]: it's using it, but pass by uikit, we shout use it directly when we modify the `Card` to be 100% antd

**Pull request merged**